sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseFacade.java

```diff
@@ -165,7 +165,7 @@ Long countCasesForMap(
 
 	FollowUpPeriodDto calculateFollowUpUntilDate(CaseDataDto caseDto, boolean ignoreOverwrite);
 
-	boolean isCaseEditAllowed(String caseUuid);
+	boolean isCaseEditAllowed(String caseUuid, boolean withArchive);
```

Comment:
@sergiupacurariu Why do we need this parameter? Shoudn't this always be checked?

Reply:
@MartinWahnschaffe I've already talked to Sergiu about this parameter and proposed an alterantive solution that he'll implement :)